docs/channels/using_channels/functions.md

```diff
@@ -78,7 +91,68 @@ exports.handler = async (pusher) => {
 };
 ```
 
-## Deploy a function
+If you are using the Pusher CLI, skip to [Functions in Pusher CLI](https://pusher.com/docs/channels/using_channels/functions/#functions-in-pusher-cli).
+
+If you are using the Pusher dashboard, when you’re finished writing your function, compress it into a .zip file making sure that the `index.js` file is in the root level, and then upload it via the dashboard following the steps below.
+
+## Functions in the Pusher dashboard
+
+1. To get started with Pusher Channels, [sign in](https://dashboard.pusher.com/accounts/sign_in) to your Pusher account. If you don’t have one, [sign up](https://dashboard.pusher.com/accounts/sign_up) for a free account.
+2. Choose the Channels app you wish to add functions to.
```

Comment:
Since we're explaining how to sign up, is it reasonable to explain how to create a new app?